pkg/network/node/ovscontroller.go

```diff
@@ -482,6 +482,12 @@ func (oc *ovsController) setupPodFlows(ofport int, podIP net.IP, vnid uint32) er
 	// IP traffic to container
 	otx.AddFlow("table=70, priority=100, ip, nw_dst=%s, actions=load:%d->NXM_NX_REG1[], load:%d->NXM_NX_REG2[], goto_table:80", ipstr, vnid, ofport)
 
+	// Isolate this pod from traffic until existing networkPolicy rules are applied
+	if oc.pluginId == 2 {
```

Comment:
ew, I don't like having ovscontroller encode this knowledge.

Can you add a `SetUpPod` API to the `osdnPolicy` interface? It can be a no-op for subnet and multitenant, and then for networkpolicy it can add these blocking rules. Then you can call `m.policy.SetUpPod` from `setup` in `pod.go` after calling `m.ovs.SetUpPod`. (Or maybe before?)

Reply:
DONE: I think that I setup as you intended with this comment